[BB2-1156] Create How TO Use Docs For BB2 SDK python

JFU-GIT opened the pull request:
<!--
You've got a Pull Request you want to submit? Awesome!
This PR template is here to help ensure you're setup for success:
  please fill it out to help ensure that your PR is complete and ready for approval.
-->

**JIRA Ticket:**
[SOMEPROJECT-42](https://jira.cms.gov/browse/SOMEPROJECT-42)

**User Story or Bug Summary:**

<!-- Please copy-paste the brief user story or bug description that this PR is intended to address. -->

### What Does This PR Do?

<!--
Add detailed description & discussion of changes here.
The contents of this section should be used as your commit message (unless you merge the PR via a merge commit, of course).

Please follow standard Git commit message guidelines:
* First line should be a capitalized, short (50 chars or less) summary.
* The rest of the message should be in standard Markdown format, wrapped to 72 characters.
* Describe your changes in imperative mood, e.g. "make xyzzy do frotz" instead of "[This patch] makes xyzzy do frotz" or "[I] changed xyzzy to do frotz", as if you are giving orders to the codebase to change its behavior.
* List all relevant Jira issue keys, one per line at the end of the message, per: <https://confluence.atlassian.com/jirasoftwarecloud/processing-issues-with-smart-commits-788960027.html>.

Reference: <https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project>.
-->

### What Should Reviewers Watch For?

<!--
Add some items to the following list, or remove the entire section if it doesn't apply for some reason.

Common items include:
* Is this likely to address the goals expressed in the user story?
* Are any additional documentation updates needed?
* Are there any unhandled and/or untested edge cases you can think of?
* Is user input properly sanitized & handled?
* Does this make any backwards-incompatible changes that might break end user clients?
* Can you find any bugs if you run the code locally and test it manually?
-->

If you're reviewing this PR, please check these things, in particular:

- TODO

dtisza1 reviewed (commented):
@JFU-GIT I pushed up some editing changes.

I added some stuff learned from the work of utilizing the SDK in the sample app also.

I moved the how-to file as the main `README.md` and moved the SDK internal development type docs to the `README-sdk-dev.md` file.

JFU-GIT commented:
> @JFU-GIT I pushed up some editing changes.
> 
> I added some stuff learned from the work of utilizing the SDK in the sample app also.
> 
> I moved the how-to file as the main `README.md` and moved the SDK internal development type docs to the `README-sdk-dev.md` file.

the refactor sounds good to me.

dtisza1 reviewed (approved):
This looks good to me!

oragame reviewed (approved):
Nice work! LGTM
